frontend/views/components/TextWho.vue

```diff
@@ -2,12 +2,11 @@
 span
   | {{firstWho}}
   template(v-if='hasWhoElse')
+    | &nbsp
     i18n and
+    | &nbsp
     tooltip
-      // TODO: replace button with i18n(tag='button' :args='{numOthers: notFirstWho.length}') {numOthers} others
-      button.is-unstyled.is-link-inherit(type='button')
-        | {{notFirstWho.length}}
-        i18n others
+      i18n.link(:args='{count: notFirstWho.length}') {count} others
```

Comment:
i18n must be a whole sentence to preserve context (individual words are never translated unless they are the entire sentence themselves, as in the "OK" in an "OK" button). Try using `html` if you want to include components within the text

Reply:
You may need to be creative here... since I don't think i18n works with `template` tags... so maybe a computed property using `L` function ... not sure

Reply:
I tried that, it didn't work... I'll leave a todo

Reply:
I can't approve this until a working version is pushed :-\